### ISSUE TITLE
 `man htop` return nothing for htop installed as snap

### ISSUE THREAD

**Issue by N0rbert** [maxiberta/htop-snap#11]
Steps to reproduce:
1. Have Ubuntu 20.04 LTS installed, without deb-version of htop
2. Run `snap install htop`, wait it to finish
3. Run `man htop`

Expected results:
* `man htop` shows man-page

Actual results:
* `man htop` does not show man-page

Bug at launchpad: https://bugs.launchpad.net/ubuntu/+source/snap/+bug/1910790 .

**Comment by maxiberta** [comment]
Indeed, the lack of man pages support in snaps can be quite annoying. Unfortunately, individual snaps can do nothing about it, as it needs to be supported by [`snapd`](https://github.com/snapcore/snapd/) ([here's the bug report](https://bugs.launchpad.net/snapd/+bug/1575593)). The good news is that [a fix has been discussed and designed at the snapcraft forum](https://forum.snapcraft.io/t/support-for-man-pages/2299). Edit: there's actually [an old PR under review](https://github.com/snapcore/snapd/pull/8079), but it's blocked :-(.

Will leave this issue open as a reminder to add a proper man page whenever the feature lands in snapd.

Thanks for your report!

**Comment by ameinild** [comment]
Maxiberta, I don't quite agree with your sentiment.

You could put the manpages inside the snap in /snap/htop/current/share/man
A user can then do the following in .bashrc (or another file): 
export MANPATH=":/snap/htop/current/share/man"

And then the manpages work. But since you didn't even include the manpages inside the snap, this workaround does not work. The workaround works on other snaps, like nano, tmux etc.

**Comment by maxiberta** [comment]
@ameinild fyi, the man page is included in the snap: `/snap/htop/current/usr/local/share/man/man1/htop.1`.

**Comment by ameinild** [comment]
Ok thanks for clearing that up.
